pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -7,18 +7,425 @@
+                        epoch_var_atts['DELTA_PLUS_VAR'])
+                    delta_plus_var_att = cdf_file.varattsget(
+                        epoch_var_atts['DELTA_PLUS_VAR'])
+
+                    # Check if a conversion to seconds is required
+                    if 'SI_CONVERSION' in delta_plus_var_att:
+                        si_conv = delta_plus_var_att['SI_CONVERSION']
+                        delta_plus_var = delta_plus_var.astype(float) \
+                            * np.float(si_conv.split('>')[0])
+                    elif 'SI_CONV' in delta_plus_var_att:
+                        si_conv = delta_plus_var_att['SI_CONV']
+                        delta_plus_var = delta_plus_var.astype(float) \
+                            * np.float(si_conv.split('>')[0])
+
+                if 'DELTA_MINUS_VAR' in epoch_var_atts:
+                    delta_minus_var = cdf_file.varget(
+                        epoch_var_atts['DELTA_MINUS_VAR'])
+                    delta_minus_var_att = cdf_file.varattsget(
+                        epoch_var_atts['DELTA_MINUS_VAR'])
+
+                    # Check if a conversion to seconds is required
+                    if 'SI_CONVERSION' in delta_minus_var_att:
+                        si_conv = delta_minus_var_att['SI_CONVERSION']
+                        delta_minus_var = \
+                            delta_minus_var.astype(float) \
+                            * np.float(si_conv.split('>')[0])
+                    elif 'SI_CONV' in delta_minus_var_att:
+                        si_conv = delta_minus_var_att['SI_CONV']
+                        delta_minus_var = \
+                            delta_minus_var.astype(float) \
+                            * np.float(si_conv.split('>')[0])
+
+                # Sometimes these are specified as arrays
+                if isinstance(delta_plus_var, np.ndarray) \
+                        and isinstance(delta_minus_var, np.ndarray):
+                    delta_time = (delta_plus_var
+                                  - delta_minus_var) / 2.0
+                else:  # And sometimes constants
+                    if delta_plus_var != 0.0 or delta_minus_var != 0.0:
+                        delta_time = (delta_plus_var
+                                      - delta_minus_var) / 2.0
+
+        if self.get_dependency(x_axis_var) is None:
+            if ('CDF_TIME' in data_type_description) or \
+                    ('CDF_EPOCH' in data_type_description):
+                xdata = cdflib.cdfepoch.unixtime(xdata)
+                xdata = np.array(xdata) + delta_time
+                if self._datetime:
+                    xdata = pds.to_datetime(xdata, unit='s')
+                self.set_dependency(x_axis_var, xdata)
+
+    def get_index(self, variable_name):
+        """Return index of variable
+
+        Parameter
+        ---------
+        variable_name : string
+
+        Returns
+        -------
+        index_ : Dependency or pandas.MultiIndex
```

Comment:
```suggestion
        Parameters
        ----------
        variable_name : str
            String to access desired data variable

        Returns
        -------
        index_ : Dependency or pandas.MultiIndex
            Index used to select desired data
```

Dependency isn't an object type, use the object type.  Also my description may not be right.